site/docs/advanced/structuring.md

```diff
@@ -20,7 +20,7 @@ That being said, a straightforward and proven strategy to structure your code is
    Every single one of these parts exposes middleware that will handle the designated messages.
 2. Create a bot instance centrally that merges all middleware by installing it onto the bot.
 3. (Optional.) Pre-filter the updates centrally, and send down updates the right way only.
-   You may also want to check out `bot.route` ([API Reference](https://doc.deno.land/https://deno.land/x/grammy/mod.ts/~/Composer#route)) or alternatively the [router plugin](../plugins/router.md) for that.
+   You may also want to check out `bot.route` ([API Reference](https://deno.land/x/grammy/mod.ts?s=Composer#method_route_0)) or alternatively the [router plugin](../plugins/router.md) for that.
```

Comment:
This link appears to be broken, so I made some adjustments to it. Could you please verify that the new one is correct?

Reply:
It is correct.